src/main/webapp/app/shared/components/atoms/tag/tag.component.scss

```
@@ -0,0 +1,20 @@
+::ng-deep .custom-tag {
```

::ng-deep is deprecated, so I would suggest not using it. I achieved the same functionality of ng-deep by adding the following line to my @Component: "encapsulation: ViewEncapsulation.None," and importing ViewEncapsulation from '@angular/core' in my atomic component.
```suggestion
.custom-tag {
```